index.html

```diff
@@ -12391,6 +12391,9 @@ <h2>Context Definitions</h2>
     <p>If the <a>expanded term definition</a> contains the <code>@language</code> <a>keyword</a>,
       its value MUST have the <a data-cite="BCP47#section-2.1.1">lexical form</a> described in [[BCP47]] or be <a>null</a>.</p>
 
+    <p class="changed">If the <a>expanded term definition</a> contains the `@index`
+      <a>keyword</a>, its value MUST be <code>true</code> or <code>false</code>.</p>
```

Comment:
I may be missing something, but I thought the value of `@index` must be a string (that resolves to an IRI)...

Reply:
Indeed, I’m not sure what I was thinking :P

Reply:
Fixed in 8d3d281085220e9825afb419a809359d9380354e.